src/test/regress/sql/babel_5343_returning.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE toverflow(id SERIAL, col1 VARCHAR);
+INSERT INTO toverflow VALUES (default, 'hope') RETURNING xmax::text::int;
+INSERT INTO toverflow VALUES (default, 'jack') RETURNING xmax;
+SELECT * FROM toverflow;
+DROP TABLE toverflow;
```

Comment:
not sure if it's worth creating a new test file just for this. Do we have any existing test file that alredy does INSERT RETURNING, if not, can probably add in insert testcase instead. 

Also, any chance we can test this code in BBF? I know we cannot really query for `xmax` etc in BBF, but wondering if there is a way to validate this in BBF

Reply:
also, `babel_5343_returning` may not make sense so much in the community. perhaps call it `insert_returning.sql`?

Reply:
The function call for BBF should've been covered in the initial test cases. Also, there is no `RETURNING` or `xmax` supported for BBF.

Actually, `insert_returning.sql` sounds like the general test cases for the `INSERT ... RETURNING` feature. But this test case is only needed due to our inappropriate handles on the codes. I prefer a specific name. After we cherry-pick, it's more consistent with the naming of similar cases in the internal repository. It doesn't matter too much, since it's not the official community repository.